app/services/request_referer_service.rb

```diff
@@ -23,6 +23,8 @@ def self.access_whitelisted?(request, controller_name, action_name)
     'host'                     => %w(show:id show_list), # http://localhost:3000/host/show/400r6
     'miq_request'              => %w(show:id show_list), # http://localhost:3000/miq_request/show/400r3
     'ems_cluster'              => %w(show:id show_list), # http://localhost:3000/ems_cluster/show/400r2
+    'ems_cloud'                => %w(show:id show_list),
+    'ems_infra'                => %w(show:id show_list),
```

Comment:
@martinpovolny I don't know deeper intention of the ENTRY_POINTS_FLAT (only what's written as comment). Is others also wrong? I see that your PR (based on my one), that was merged, is also whitelist new resources. Why did you leave it there if it's wrong in your opinion?